doc/gui/examples/controls/chat-discuss.py

```diff
@@ -24,7 +24,7 @@
 from taipy.gui.gui_actions import navigate, notify
 
 username = ""
-users: list[str|Icon] = []
+users: list[str | Icon] = []
```

Comment:
Is this supported in python 3.8 ?

Reply:
I just double-check it and it's not. @FabienLelaquais What's your thought on this

Reply:
My thoughts:
all this comes from the generation of the Page Builder API, in taipy/gui/builder/__init__.pyi.
I want (see #1632) the property types and default values to be injected automatically in the function signature.
The 'optional' syntax forces to import 'types' is is meant to disapear.

Now.
If it does not work, well it does not. So I have to step back at this point.
That will be part of the IDE autocompletion issue.
Thanks.